cr-examples/triton/src/test/java/oracle/code/triton/TritonTestExtension.java

```diff
@@ -107,6 +111,17 @@ void test(CoreOp.FuncOp javaKernel,
                 return TritonTransformer.tritonModule(javaKernel, JavaType.VOID, argTypes);
             });
 
+            String mlirText = MLIRGenerator.transform(actualTritonKernel);
```

Comment:
We should write the files to the maven target directory. Add the following to the surefire plugin configuration:
```
                        <systemPropertyVariables>
                            <project.build.directory>${project.build.directory}</project.build.directory>
                        </systemPropertyVariables>
```
Then replace the code below with:
```
            Path buildDir = Path.of(System.getProperty("project.build.directory", ""));
            Path mlirDir = buildDir.resolve("mlir");
            try {
                Files.createDirectories(mlirDir);
                Files.writeString(mlirDir.resolve(javaKernelName + ".mlir"), mlirText, StandardOpenOption.CREATE);
            } catch (IOException e) {
                e.printStackTrace();
            }
```

Reply:
Thanks, fixed.